_events/2022/2022-01-community-call.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Jan 2022 Community Call
+expires: 2022-01-13
```

Comment:
I'm sorry, I forgot: did we establish that same-day expiry works okay?

Reply:
I think we would likely want it to expire the day after. It also totally depends on when the site is updated, as the rendering is relevant to that date. So if it were the case that the site was redeployed on the 13th, this wouldn't shot up (and I suspect we'd want it to).

Reply:
And if we haven't done this in the past, likely we've just lucked out to not have it render on that same day! :laughing:

Reply:
```suggestion
expires: 2022-01-14
```